war/src/main/scss/abstracts/colors.scss

```diff
@@ -0,0 +1,44 @@
+@use "theme";
+
+// Generates a series of color override classes and their variations,
+// e.g. .jenkins-!-color-blue, .jenkins-!-color-light-blue, .jenkins-!-color-dark-blue
+@each $key in theme.$colors {
+  .jenkins-\!-color-light-#{key} {
```

Comment:
Something seems to be going wrong here. The generated CSS file in your new version has `.jenkins-\!-color-light-key` rather than e.g. `.jenkins-\!-color-light-brown` as expected.

Reply:
Great spot, fixed.